tutorial.md

```diff
@@ -104,6 +104,9 @@ go version go1.8.5 linux/amd64
 
 ```
 go get github.com/kubernetes-incubator/cri-tools/cmd/crictl
+cd ./src/github.com/kubernetes-incubator/cri-tools/cmd/crictl
+make 
```

Comment:
Don't we need a specific crictl commit? I remember running into this issue a few days back unless cri-o now works with the latest crictl.

Reply:
We should point users to the crictl that we have in our tests or Dockerfile. So they don't mismatch the versions.

Reply:
> We should point users to the crictl that we have in our tests or Dockerfile…

Can we get them to cut releases?  Then we might be able to `go get gopkg.in/kubernetes-incubator/cri-tools.v2` or similar using [gopkg.in's supported URIs][1].  On the other hand, I'm not sure if gopkg.in can handle the `cmd/crictl` nesting…

[1]: http://labix.org/gopkg.in#SupportedURLs

Reply:
Yeah, we should open an issue to start cutting releases for cri-tools.